src/Components/Resource/BadgesList.tsx

```diff
@@ -75,13 +41,21 @@ export default function BadgesList(props: any) {
         }),
         ...dateRange("Modified", "modified_date"),
         ...dateRange("Created", "created_date"),
-        value("Origin facility", "origin_facility", orginFacilityName),
+        value(
+          "Origin facility",
+          "origin_facility",
+          originFacilityData?.data?.name
```

Comment:
Use proper variable names. `data.data` doesn't sound right right?
```suggestion
          originFacility?.data?.name
```